laynebritton/jl-fast-treecapitator
# DFS Optimizations

- Reducing the amount of recursive calls by checking visited before recursively calling DFS on a block
- Moved destroy to a util helper function

Testing

- Verify a super tall jungle tree and dark oak tree can be vein mined
- Verify an ore vein can be vein mined
